src/Formatter/JsonFormatter.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Roave\BackwardCompatibility\Formatter;
+
+use Psl\Str;
+use Roave\BackwardCompatibility\Changes;
+use Roave\BackwardCompatibility\Formatter\OutputFormatter;
+use Roave\BackwardCompatibility\Git\CheckedOutRepository;
+use Symfony\Component\Console\Output\OutputInterface;
+use function Psl\Str;
+
+/** @internal */
+final class JsonFormatter implements OutputFormatter
```

Comment:
We should ship an accompanying JSON Schema file with this, if we want consumers downstream to rely on it. It suffices for it to be in the docs, although verifying the schema in the test suite would be preferable.